lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -37,9 +38,8 @@ def initialize(file)
       @file = file = File.expand_path(file)
       raise KafoParsers::ModuleName, "File not found #{file}, check your answer file" unless File.exist?(file)
 
-      command = "#{self.class.puppet_bin} strings generate --format json #{file}"
-      @raw_json = `#{command}`
-      unless $?.success?
+      @raw_json, status = capture2("#{self.class.puppet_bin} strings generate --format json #{file}")
```

Comment:
Perhaps while we're at it, it should become an array to avoid dealing with escaping, if needed?
```suggestion
      @raw_json, status = capture2([self.class.puppet_bin, 'strings', 'generate', '--format', 'json', file])
```